.rubocop.yml

```diff
@@ -2,6 +2,11 @@
 require: rubocop-rspec
 inherit_from: .rubocop_todo.yml
 
+# With the rubocop 0.47.0 and rubocop-rspec 1.8.0 the following stderr message was sent:
+#   An error occurred while RSpec/DescribedClass cop was inspecting path/to/file
```

Comment:
I think the issue is that 0.47.0 was released and 1.10.0 of rubocop-rspec was released. So we were running into conflicts in API changes (or some such thing).